cmd/launcher/watchdog/watchdog_service_windows.go

```diff
@@ -0,0 +1,231 @@
+//go:build windows
+// +build windows
+
+package watchdog
+
+import (
+	"context"
+	"fmt"
+	"log/slog"
+	"os"
+	"time"
+
+	"github.com/kolide/kit/version"
+	agentsqlite "github.com/kolide/launcher/ee/agent/storage/sqlite"
+	"github.com/kolide/launcher/ee/gowrapper"
+	"github.com/kolide/launcher/pkg/launcher"
+	"github.com/kolide/launcher/pkg/log/multislogger"
+	"github.com/kolide/launcher/pkg/log/sqlitelogger"
+	"github.com/pkg/errors"
+	"golang.org/x/sys/windows"
+	"golang.org/x/sys/windows/svc"
+	"golang.org/x/sys/windows/svc/mgr"
+)
+
+type winWatchdogSvc struct {
+	systemSlogger, slogger *multislogger.MultiSlogger
```

Comment:
There's a lot here that feels out of place in the system logs -- These are supposed to land in the event logs and be customer facing. Simple start and stop make sense. But all the details, and stack traces... Though could go straight to the `ll` logger and no where else.